runbotics/runbotics-common/src/actions/actions.ts

```diff
@@ -204,7 +205,8 @@ export enum ExcelAction {
     DELETE_ROWS = 'excel.deleteRows',
     SAVE = 'excel.save',
     CLOSE = 'excel.close',
-    EXPORT_TO_CSV = 'excel.exportToCsv'
+    EXPORT_TO_CSV = 'excel.exportToCsv',
+    RANGE_TO_HTML_TABLE = 'excel.rangeToHtmlTable'
```

Comment:
Imo it is better to call it "Export HTML table". The `range` is not important at the beginning, the user will find out what is required to be passed to the action when he/she goes to the properties